lib/number.tsx

```diff
@@ -0,0 +1,60 @@
+import { ReactNode } from "react"
+
+/**
+ * Styles non-numeric characters in a given string by wrapping them in a span with passed className applied.
+ * Numeric characters are returned as-is.
+ *
+ * @param numString - The string containing numeric and non-numeric characters.
+ * @param className - Styling class names to apply to the span wrapping non-numeric characters.
+ * @returns An array of ReactNode elements where non-numeric characters are wrapped in a span with
+ *          the classes provided.
+ */
+export const stylePunctuation = (
```

Comment:
Don't love this change. The idea of `lib` is to separate concerns. This could easily be controlled inside a UI component.

```tsx
<NumberHidePunctuation number={gas} />
```

Reply:
@pettinarip Yeah good call, was thinking about that as I updated to .tsx... can refactor as such